pkg/framework/deployment.go

```diff
@@ -61,7 +78,12 @@ func UpdateDeployment(c client.Client, name, namespace string, updated *kappsapi
 
 // IsDeploymentAvailable returns true if the deployment has one or more availabe replicas
 func IsDeploymentAvailable(c client.Client, name, namespace string) bool {
-	if err := wait.PollImmediate(RetryShort, WaitLong, func() (bool, error) {
+	return IsDeploymentAvailableIn(c, name, namespace, WaitLong)
```

Comment:
why do we need to wrap this and have two different functions? is there any valid case where we want to use different wait timeout here?